uber/config.py

```diff
@@ -486,6 +522,7 @@ def _unrepr(d):
         c.PRICE_LIMITS[int(_opt)] = _val
     else:
         c.PRICE_BUMPS[date] = _val
+c.ORDERED_PRICE_LIMITS = sorted([val for key, val in c.PRICE_LIMITS.items()])
```

Comment:
I used the values here because that lets me do this:
`current_price_tier = c.ORDERED_PRICE_LIMITS.index(c.BADGE_PRICE) if c.BADGE_PRICE in c.ORDERED_PRICE_LIMITS else -1`

If I don't have access to the values, it's harder to find what the current price tier is, so it's kind of 6 on one hand, half-dozen on the other.

Of course, if we can have both things easily, that'd be awesome! But I started trying to do that and, man, python iterative objects get really weird with being unordered or not being able to find the index based on value or not being able to just step from one key to the next. Let me know if you have any ideas for what the best data-type might be here!